upt_macports/templates/base.Portfile

```diff
@@ -4,16 +4,22 @@ PortSystem          1.0
 {% block portgroup %}
 {% endblock %}
 
+# Portfile generated/maintained using the Universal Packaging Tool (upt)
```

Comment:
I would probably use `maintained` only after we get automatic updates working :)

Reply:
we are optimistic about it ;)